### PR TITLE
Khp3 6325 filter out family relations in pns contacts leaving only sexual relationship

### PR DESCRIPTION
**Pull request by Omoshlawi**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
https://thepalladiumgroup.atlassian.net/browse/KHP3-6325

## Screenshots
image
Screenshot from 2024-08-07 09-36-16

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-lab-manifest-app/src/forms/lab-manifest-form.workspace.tsx [hunk 1]
```diff
@@ -61,8 +61,9 @@ const LabManifestForm: React.FC<LabManifestFormProps> = ({ closeWorkspace, manif
   };
   return (
     <Form onSubmit={form.handleSubmit(onSubmit)}>
-      <span className={styles.formTitle}>{t('formTitle', 'Fill in the form details')}</span>
       <Stack gap={4} className={styles.grid}>
+        <span className={styles.sectionHeader}>Manifest Date range</span>
```
Comment (donaldkibet) [inline packages/esm-lab-manifest-app/src/forms/lab-manifest-form.workspace.tsx:65]: Lets just maintain the convention by having these as translated translation e.g
```tsx
t('manifestDateRange', 'Manifest Date range')
```

File: packages/esm-lab-manifest-app/src/header/lab-manifest-header.scss [hunk 2]
```diff
@@ -101,24 +102,30 @@ svg.iconOverrides {
...
   margin-bottom: spacing.$spacing-05;
-
   flex-flow: row wrap;
   align-items: center;
 }
 
+.btnSetRight {
+  display: flex;
+  flex-direction: row;
+  // gap: spacing.$spacing-05;
```
Comment (donaldkibet) [inline packages/esm-lab-manifest-app/src/header/lab-manifest-header.scss:112]:
```suggestion
 
```

File: packages/esm-lab-manifest-app/src/tables/filter-table-header.component.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+interface FilterTableHeaderProps {
+  searchText?: string;
+  onSearch?: (string: string) => void;
+}
+
+const FilterTableHeader: React.FC<FilterTableHeaderProps> = ({ onSearch, searchText = '' }) => {
+  return (
+    <div>
+      <Search
+        placeholder="Search this table"
```
Comment (donaldkibet) [inline packages/esm-lab-manifest-app/src/tables/filter-table-header.component.tsx:14]: translation

**Review by donaldkibet — CHANGES_REQUESTED** [review]
Thanks @Omoshlawi, seen changes on lab manifest yet the ticket indicates work on relationship. It would be good to separate the concern into two PRs. Thanks! 🍻

**Comment by Omoshlawi** [comment]
@donaldkibet  , made 2 PRS for both 
Lab manifest: https://github.com/palladiumkenya/kenyaemr-esm-3.x/pull/297 , however it hasen't been merged, So I meged its branch locally tow work on this ticket branch since I needed some changes on it.

**Review by donaldkibet — APPROVED** [review]
Thanks @Omoshlawi LGTM

**Pull request merged**